se/se_epub_lint.py

```diff
@@ -672,6 +687,12 @@ def lint(self, skip_lint_ignore: bool, allowed_messages: List[str] = None) -> li
 
 	# Done parsing ignore list
 
+	# Sanity check the spine as we’ll be using it later
```

Comment:
Shouldn't this be a lint error code instead of a fatal error that crashes lint? I don't think a file missing from the spine means the ebook is totally invalid - I think that's a valid state for an epub to be in

Reply:
OK, makes sense.